README.md

```diff
@@ -1,7 +1,7 @@
 <a href="https://engineering.carrot.is/"><p align="center"><img src="https://cloud.githubusercontent.com/assets/2105067/24525319/d3d26516-1567-11e7-9506-7611b3287d53.png" alt="Go Carrot" width="350px" align="center;" /></p></a>
 # Nomo
 
-Nomo is a library that converts [go-carrot/turf](https://github.com/go-carrot/turf) controllers into [Swagger/OpenAPI Specification (v3.0.0)](https://swagger.io/specification).
+Nomo is a library that converts [go-carrot/turf](https://github.com/go-carrot/turf) controllers into [Swagger/OpenAPI Specification (v2.0)](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
```

Comment:
Switching to 2.0, currently because there is more support for tools in the community right now.

Will upgrade to 3.0 in the future.